doc/architectural-overview.md

```diff
@@ -108,33 +108,23 @@ different set of workers to a machine agent.
 There are a number of *clients* which interact with Juju using the Juju
 API. These include the `juju` command line tool and Juju Dashboard.
 
-
-## The Data Store (aka "state")
+## The Domain
 
 There's a lot of *detail* to cover, but there's not much to say from an architectural
-standpoint. We use a mongodb replicaset to support HA; we use the `mgo` package from
-`labix.org` to implement multi-document transactions; we make use of the transaction
-log to detect changes to particular documents, and convert them into business-object-
-level events that get sent over the API to interested parties.
+standpoint. We use DQLite to store the domain state and support HA.
+
+The domain package consists of several component package, stored in two SQL schema,
+one for the controller level, another for the model level. Each component provides a
+state layer to deal with the database and a service layer to provide access to client.
```

Comment:
several ... packages* 

two SQL schemas*

and one* for the model level

to the* client

Reply:
Thanks, spellfix done !